assets/js/googlesitekit/datastore/site/urls.js

```diff
@@ -24,6 +24,12 @@ import { CORE_SITE } from './constants';
 import { getLocale } from '../../../util';
 const { createRegistrySelector } = Data;
 
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+import { isNumber } from 'lodash';
```

Comment:
Can you load only the part of the library you need individually? Please refer to the[ 10up Engineering best practices docs](https://10up.github.io/Engineering-Best-Practices/javascript/#performance).
```suggestion
import isNumber from 'lodash/isNumber';
```